### ISSUE TITLE
Please help , trying to understand when compute_loss is called .

### ISSUE THREAD

**Issue by stavBodik** [keras-team/keras#8345]
I am trying to debug compute_loss function ,
In my code the loss function :

def compute_loss(self, y_true, y_pred): """Compute mutlibox loss......

Is passed to the compile function :

model.compile(optimizer=optimizer, loss=MultiboxLoss(NUM_CLASSES, alpha=MultiboxLoss_alpha, neg_pos_ratio=neg_pos_ratio, negatives_for_hard=MultiboxLoss_negatives_for_hard, class_weight=class_weight_train).compute_loss)

Then as I understand right , this function is callable , and should be called inside the training function:

model.fit_generator(generator=gen_train.generate(True),....

where can I find the lines which is executing the call to this function ?
I cant stop there with debugger unfortunately because its callable function , like pointer to function in memory .

Please help ? Thanks !

**Comment by ozabluda** [comment]
I think it's called inside the backend (say TF), not anywhere in Keras.

**Comment by stavBodik** [comment]
Yes your right but I cant find way to debug it , I want to see which x,y inputs it gets , anyone please ?
